main.tf

```diff
@@ -0,0 +1,136 @@
+resource "aws_lb" "api" {
+  name_prefix        = "apilb-"
+  internal           = false
+  load_balancer_type = "application"
+  security_groups    = [aws_security_group.lb.id]
+  subnets            = var.subnets
+
+  enable_deletion_protection = false
+}
+
+resource "aws_lb_target_group" "api_health_check" {
+  name_prefix = "hlt-"
+  port        = 8080
+  protocol    = "HTTP"
+  vpc_id      = var.vpc_id
+
+  health_check {
+    healthy_threshold   = 2
+    unhealthy_threshold = 2
+    timeout             = 3
+    path                = "/healthz"
+    port                = 8080
+    interval            = 30
+  }
+}
+
+resource "aws_lb_target_group" "external_api" {
+  name_prefix = "ext-"
+  port        = 3013
+  protocol    = "HTTP"
+  vpc_id      = var.vpc_id
+
+  health_check {
+    healthy_threshold   = 2
+    unhealthy_threshold = 2
+    timeout             = 3
+    path                = "/healthz"
+    port                = 8080
+    interval            = 30
+  }
+}
+
+resource "aws_lb_target_group" "internal_api" {
+  count       = var.internal_api_enabled ? 1 : 0
+  name_prefix = "int-"
+  port        = 3113
+  protocol    = "HTTP"
+  vpc_id      = var.vpc_id
+
+  health_check {
+    healthy_threshold   = 2
+    unhealthy_threshold = 2
+    timeout             = 3
+    path                = "/healthz"
+    port                = 8080
+    interval            = 30
+  }
+}
+
+resource "aws_lb_target_group" "state_channels_api" {
+  count       = var.state_channel_api_enabled ? 1 : 0
+  name_prefix = "sc-"
+  port        = 3014
+  protocol    = "HTTP"
+  vpc_id      = var.vpc_id
+
+  health_check {
+    healthy_threshold   = 2
+    unhealthy_threshold = 2
+    timeout             = 3
+    path                = "/healthz"
+    port                = 8080
+    interval            = 30
+  }
+
+  stickiness {
+    enabled         = true
+    type            = "lb_cookie"
+    cookie_duration = 86400
+  }
+}
+
+resource "aws_alb_listener" "api" {
+  load_balancer_arn = aws_lb.api.arn
+  port              = 80
+  protocol          = "HTTP"
+
+  default_action {
+    type             = "forward"
+    target_group_arn = aws_lb_target_group.external_api.arn
+  }
+}
+
+resource "aws_lb_listener_rule" "health_check" {
+  listener_arn = "${aws_alb_listener.api.arn}"
```

Comment:
i guess `${` syntax is no longer needed

Reply:
it's not, but it's harmless as well, I copy paster a lot of the configuration and don't want to spend time removing this